saleor/graphql/webhook/mutations.py

```diff
@@ -132,6 +139,16 @@ class Meta:
         error_type_class = WebhookError
         error_type_field = "webhook_errors"
 
+    @classmethod
+    def clean_input(cls, info, instance, data, input_cls=None):
+        cleaned_input = super().clean_input(info, instance, data, input_cls)
+        # This should be removed after we drop service_account
```

Comment:
Please open an issue to track this

Reply:
I already did that https://github.com/mirumee/saleor/issues/5521 ;)